src/eos-updater-apply.c

```diff
@@ -167,6 +167,14 @@ apply_internal (EosUpdater *updater,
   g_clear_object (&data->extensions);
   g_clear_error (&local_error);
 
+  /* Cleaning up after update should be non-fatal, since we've already
+   * successfully deployed the new OS. This clearly is a workaround
+   * for a more serious issue, likely related to concurrent prunes. */
```

Comment:
I’d like to see a `FIXME` and a reference to the Phab task for fixing concurrent prunes here.